pom.xml

```diff
@@ -413,4 +413,18 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>github</id>
+            <distributionManagement>
+                <repository>
+                    <id>github</id>
+                    <name>github-maven-repository</name>
+                    <url>https://maven.pkg.github.com/apache/skywalking-banyandb-java-client</url>
+                </repository>
+            </distributionManagement>
+        </profile>
+    </profiles>
```

Comment:
Remove this, I mentioned using this on the main repo, not here. Or you could keep on ASF snapshot.

Reply:
Sure. Removed